lib/internal/repl/utils.js

```diff
@@ -114,7 +114,7 @@ function isRecoverableError(e, code) {
   // Try to parse the code with acorn.  If the parse fails, ignore the acorn
   // error and return the recoverable status.
   try {
-    RecoverableParser.parse(code, { ecmaVersion: 11 });
+    RecoverableParser.parse(code, { ecmaVersion: 2021 });
```

Comment:
```suggestion
    RecoverableParser.parse(code, { ecmaVersion: 'latest' });
```

Reply:
> 
> 
> I suggest to use `latest` instead to prevent future updates. This is a recent feature added to acorn (https://github.com/acornjs/acorn/tree/master/acorn#interface).

Is that not going to cause maintenance issues for release lines?